src/link-manager.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @fileoverview Link manager for controlling the link query.
+ */
+
+/** @const */
+genotet.linkManager = {};
+
+/**
+ * Object that stores the link views by their type.
+ */
+/** @type {!Object<!Array<{
+ *    target: genotet.View,
+ *    action: string,
+ *    response: string
+ *  }>>}
+ */
+genotet.linkManager.links = {};
+
+/** @const {!Array<string>} */
+genotet.linkManager.EXPRESSION_ACTIONS = ['addGeneProfile'];
+
+/** @const {!Array<string>} */
+genotet.linkManager.NETWORK_ACTIONS = ['nodeClick', 'edgeClick'];
+
+/** @const {!Array<string>} */
+genotet.linkManager.BINDING_ACTIONS = ['updateTrack', 'locus'];
+
+/** @const {string} */
+genotet.linkManager.DEFAULT_MAPPING_FILENAME = 'namecode';
```

Comment:
If there exists BATF.bw, there should be another choice in mapping setting dialog named 'No mapping file', so that the user can choose to use BATF.bw again. Otherwise, it is meaningless to set default mode to no mapping.